### ISSUE TITLE
Improve user tracking and learn how they use the product

### ISSUE THREAD

**Issue by menor** [resmio/git-along#14]
coming from [here](https://github.com/resmio/git-along/issues/6#issuecomment-247605267)

>We don't measure the success of features. Automated newsletters are being used? What about the Tags? Did they stopped to use the 'Notes' field? Etc... we never know if our deploys have good, bad or any impact on our users. Segment, Analytics, FullStory? Surveys? I don't know..
@fabiennepimenta 

We should find some service that helps us with this and implement it. Segment?

**Comment by fabiennepimenta** [comment]
Full Story is great to see how people actually use our product, and there is so many companies using it that I don't know how this legal privacy thing works. 

If you don't believe me check this list of articles about fullstory: https://medium.com/search?q=fullstory

**Comment by menor** [comment]
I believe you, but we are not the ones going to jail if that is not legal. @mmarder can you ask some one about this (full story)? I agree with Fabi it would be incredibly useful. I think we also need some analytics on top of that to see how many people uses each feature. I guess we need to do some research about solutions.

**Comment by ihucos** [comment]
I fully agree with with this point and am very happy that we are doing steps in this direction. We should also not forget to have personal relationships or feedback talks with customers to really understand them and always ask ourselves how we can create value for our customers.

**Comment by menor** [comment]
@mmarder I need your help to close this one
